include/os/linux/etcpal/os_event_group.h

```diff
@@ -27,14 +27,15 @@ extern "C" {
 #include <pthread.h>
 #include <stdbool.h>
 #include <stdint.h>
+#include "etcpal/common.h"
 
 typedef uint32_t etcpal_event_bits_t;
 
 typedef struct
 {
-  bool valid;
-  pthread_cond_t cond;
-  pthread_mutex_t mutex;
+  bool                valid;
+  pthread_cond_t      cond;
+  pthread_mutex_t     mutex;
```

Comment:
Why did the format change here?

Reply:
I assume because it was previously not auto-formatted and I had my editor set to format on save.